### PR TITLE
Variables from User-Run moved

### PR DESCRIPTION
**Pull request by KnifMelti**
# Proposed Changes

> Variables in User-Run.ps1 moved to Start-NotifTask.ps1

## Related Issues

#311

### CODE REVIEW

**Comment by KnifMelti** [comment]
@FaserF

**Comment by FaserF** [comment]
Nice one! Thanks

File: Winget-AutoUpdate/functions/Start-NotifTask.ps1 [hunk 1]
```diff
@@ -14,6 +14,13 @@ function Start-NotifTask {
         [Switch]$ButtonDismiss = $false
     )
 
+    $caller = Get-ChildItem $MyInvocation.PSCommandPath | Select-Object -Expand Name
+    if ($caller -eq "User-Run.ps1") {
+        $Title = "Winget-AutoUpdate (WAU)"
+        $Balise = "Winget-AutoUpdate (WAU)"
+        $UserRun = $True
+    }
+
```
Comment (Romanitho) [inline Winget-AutoUpdate/functions/Start-NotifTask.ps1:23]: and what about adding default values in the param ? something like :

```
function Start-NotifTask {
    param(
        [String]$Title = "Winget-AutoUpdate",
        [String]$Message,
        [String]$MessageType,
        [String]$Balise = "WAU",
        [String]$OnClickAction,
        [String]$Body,
        [String]$Button1Text,
        [String]$Button1Action,
        [Switch]$ButtonDismiss = $false
        [Switch]$UserRun = $false
    )
```

this way, `Title` and `Balise` got default values, and you just need to add `-UserRun` when you call this function from the `user-run.ps1` script

Comment (KnifMelti) [reply Winget-AutoUpdate/functions/Start-NotifTask.ps1:23]: Better

**Pull request merged**